packages/showcases/svelte/stories/components/selection-controls/VtmnCheckbox.stories.svelte

```diff
@@ -0,0 +1,23 @@
+<script>
+  import { Meta, Template, Story } from '@storybook/addon-svelte-csf';
+  import { VtmnCheckbox } from '@vtmn/svelte';
+  import { parameters } from '@vtmn/showcase-core/csf/components/selection-controls/checkbox.csf';
+
+  const checkboxArgs = {
+    id: 'My checkbox',
+    label: 'This is the label of the checkbox',
+    disabled: false,
+  };
+</script>
+
+<Meta
+  title="Components / Selection controls / VtmnCheckbox"
+  component={VtmnCheckbox}
+  {parameters}
+/>
+
+<Template let:args>
+  <VtmnCheckbox label={args.label} id={args.id} disabled={args.disabled} />
```

Comment:
If comments bellow are accepted, we need to add : 
- `checked` attribute on  the `VtmnCheckbox`
- `label` can be remove
- You can now user <VtmnCheckbox>label</VtmnCheckbox>

Reply:
Done